src/sinks/util/unix.rs

```diff
@@ -192,6 +239,31 @@ impl Sink for UnixSink {
     }
 }
 
+pub struct UnixService {
+    connector: UnixConnector,
+}
+
+impl tower::Service<Bytes> for UnixService {
+    type Response = ();
+    type Error = UnixSocketError;
+    type Future = BoxFuture<'static, Result<(), Self::Error>>;
+
+    fn poll_ready(&mut self, _: &mut Context<'_>) -> Poll<Result<(), Self::Error>> {
+        Poll::Ready(Ok(()))
+    }
+
+    fn call(&mut self, msg: Bytes) -> Self::Future {
+        use futures::SinkExt;
+        let connector = self.connector.clone();
+        async move {
+            let mut connection = connector.connect().await?;
+            connection.send(msg).await.context(SendError)?;
+            Ok(())
```

Comment:
```suggestion
            connector.connect()
                .await?
                .send(msg)
                .await
                .context(SendError)
```
(might need a `.map(|_| ())` at the end)